docs/userguide/declarative_config.rst

```diff
@@ -166,6 +166,12 @@ Special directives:
       The ``file:`` directive is sandboxed and won't reach anything outside the
       project directory (i.e. the directory containing ``setup.cfg``/``pyproject.toml``).
 
+  .. attention::
```

Comment:
@abravalheri seeing this copy-pasted 3 times, I'd suggest putting it into a separate document under some `docs/_fragments/` location, and including it from there.